addons/hr_timesheet_sheet/hr_timesheet_sheet.py

```diff
@@ -140,11 +140,10 @@ def attendance_action_change(self, cr, uid, ids, context=None):
         return hr_employee.attendance_action_change(cr, uid, employee_ids, context=context)
     
     def _count_all(self, cr, uid, ids, field_name, arg, context=None):
-        Timesheet = self.pool['hr.analytic.timesheet']
         Attendance = self.pool['hr.attendance']
         return {
             sheet_id: {
-                'timesheet_activity_count': Timesheet.search_count(cr,uid, [('sheet_id','=', sheet_id)], context=context),
+                'timesheet_activity_count': self.pool['account.analytic.line'].search_count(cr,uid, [('sheet_id','=', sheet_id)], context=context),
```

Comment:
I think this method should be rewritten using read_groups, instead of a search_count + loop. More efficient.